src/test/java/com/iexec/sms/admin/AdminServiceTests.java

```diff
@@ -179,21 +178,46 @@ void withSQLException() {
     @Test
     void shouldDeleteBackup() throws IOException {
         final String backupFileName = "backup.sql";
-        final Path tmpFile = Files.createFile(tempStorageLocation.toPath().resolve(backupFileName));
+        final File backupDatabaseFile = new File(tempStorageLocation.getPath() + "/" + backupFileName + AdminService.AES_KEY_FILENAME_EXTENSION);
```

Comment:
Error on the backup filename:
```suggestion
        final File backupDatabaseFile = new File(tempStorageLocation.getPath() + "/" + backupFileName);
```